ui/src/components/ClientForm/index.jsx

```diff
@@ -323,15 +328,13 @@ export default class ClientForm extends Component {
               }
             />
           </ListItem>
-          <ListItem>
-            <TextField
-              label="Description"
-              name="description"
+          <ListItem className={classes.clickDescriptionListItem}>
```

Comment:
What does `click` in `clickDescriptionListItem` mean? What do you think if we omit `click` and instead rename it to `descriptionListItem`.

Reply:
What do you think of changing clientDescriptionListItem? I thought about differentiating this description view from others.

Reply:
Sure. `clientDescriptionListItem` sounds good to me :)

Reply:
done!